app/_locales/en/messages.json

```diff
@@ -607,7 +607,8 @@
     "message": "If you need to back up your seed phrase again, you can find it in Settings -> Security."
   },
   "endOfFlowMessage7": {
-    "message": "If you ever have questions or see something fishy, email support@metamask.io."
+    "message": "If you ever have questions or see something fishy, contact our support $1.",
+    "description": "$1 is a clickable link."
```

Comment:
For each of these descriptions, it's probably worth mentioning what the clickable link text will say ("here"). When I first read this, I thought it meant that the URL would be passed in as $1.

Reply:
Updated descriptions here